context/app/static/js/components/Detail/visualization/Visualization/Visualization.jsx

```diff
@@ -18,21 +21,34 @@ import {
   StyledHeaderText,
   StyledHeaderRight,
   ExpandButton,
-  EscSnackbar,
+  VitessceInfoSnackbar,
   ErrorSnackbar,
   ExpandableDiv,
   StyledFooterText,
   SelectionButton,
 } from './style';
+import { useVitessceConfig } from './hooks';
 import 'vitessce/dist/es/production/static/css/index.css';
 
+function sniffBrowser() {
+  const { browser } = Bowser.parse(window.navigator.userAgent);
+  return browser.name;
+}
+
+const FIREFOX_WARNING =
+  'Visualizations may be slow on Firefox.  If there is latency, please check if you are on the latest version - upgrading Firefox may alleviate the problem.';
```

Comment:
```suggestion
  'On older versions of Firefox, zooming in on visualizations is very slow; Upgrading to the latest version will fix the problem.';
```
- "may be" -> "is"
- Be more more precise about what interactions are slow.
- A little shorter.

Thanks!